### PR TITLE
Feature/smishenk/totaltime

### PR DESCRIPTION
**Pull request by smishenk**

### CODE REVIEW

**Comment by eleskine** [comment]
Hi Sergey. First off, you will need to squash your commits into a single one. You can do it using command "git rebase -i HEAD~9" where 9 is the number of commits being squashed

File: src/profile-generator-inl.h [hunk 1]
```diff
@@ -41,9 +41,10 @@ bool CodeEntry::is_js_function_tag(Logger::LogEventsAndTags tag) {
 }
 
 
-ProfileNode::ProfileNode(ProfileTree* tree, CodeEntry* entry)
+ProfileNode::ProfileNode(ProfileTree* tree, CodeEntry* entry, int src_line /*= v8::CpuProfileNode::kNoLineNumberInfo*/)
```
Comment (eleskine) [inline src/profile-generator-inl.h]: Remove the commented out stuff.

File: src/profile-generator.cc [hunk 2]
```diff
@@ -221,20 +221,35 @@ void ProfileNode::CollectDeoptInfo(CodeEntry* entry) {
...
+    }
+    return ProfileNode::FindChild(&CallSite());
 }
 
 
-ProfileNode* ProfileNode::FindOrAddChild(CodeEntry* entry) {
-  HashMap::Entry* map_entry =
-      children_.Lookup(entry, CodeEntryHash(entry), true);
-  ProfileNode* node = reinterpret_cast<ProfileNode*>(map_entry->value);
+ProfileNode* ProfileNode::FindChild(CallSite* callsite) {  HashMap::Entry* map_entry =
```
Comment (eleskine) [inline src/profile-generator.cc]: Please format the code.

**Comment by smishenk** [comment]
I will rebase the branch when all actions and modifications will be taken

**Comment by eleskine** [comment]
LGTM

**Pull request merged**